examples/ex010_advection.jl

```diff
@@ -0,0 +1,54 @@
+# ---------------------------------------------------------------
+# advection example following Melvin, Staniforth and Thuburn
+# Q.J:R. Meteorol. Soc. 138: 1934-1947, Oct (2012)
+# non polynomial advection example using transplanted
+# quadrature formulas from conformal maps following Hale and 
+# Trefethen (2008) SIAM J. NUMER. ANAL. Vol. 46, No. 2
+# ---------------------------------------------------------------
+using LFAToolkit
+using LinearAlgebra
+using Polynomials
+
+# setup
+mesh = Mesh1D(1.0)
+P = 11;
```

Comment:
No need to be very high order in the demo. Better to run fast and be easy to interpret.
```suggestion
P = 4;
```